cloudsupport/v1/ekssupport.go

```diff
@@ -19,17 +22,40 @@ type IEKSSupport interface {
 	GetName(*eks.DescribeClusterOutput) string
 	GetRegion(cluster string) (string, error)
 	GetContextName(cluster string) string
+	GetAwsCfgMap(kapi *k8sinterface.KubernetesApi, namespace string) error
 }
 
 type EKSSupport struct {
 }
 
+// ListmapRoles will store mapped roles found inside the aws-auth
+var ListmapRoles []*MappedRoles
```

Comment:
Why are you using a global list? I think this list should be returned from the function